erts/doc/src/erlang.xml

```diff
@@ -2068,6 +2068,10 @@ end</pre>
           Typically, this is used when a process started from a
           certain shell is to have another group leader than
           <c>init</c>.</p>
+        <p>The process leader should be rarely changed in
```

Comment:
Please change "process leader" and "project leader" to "group leader". And maybe we should be a bit more firm? A note? @bjorng , what do you think? Or at least use "must not" instead of "should rarely"?
No details must be written here, but please add link to the design principles (use `seealso` tag, with attribute `marker="doc/design_principles:applications#stopping"`).